sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -627,8 +644,32 @@ && getConnectionInitSqls().get() != null
         if (getMaxConnections() != null && getMaxConnections().get() != null) {
           basicDataSource.setMaxTotal(getMaxConnections().get());
         }
-        if (getDriverClassLoader() != null) {
-          basicDataSource.setDriverClassLoader(getDriverClassLoader());
+        if (getDriverJarDirectory() != null && getDriverJarDirectory().get() != null) {
+          try {
+            String pathPrefix =
+                getDriverJarDirectory().get().endsWith("/")
```

Comment:
Hardcoding "/" will break this for windows. Please use System.lineSeparator()  instead. Also please make this a util instead of directly including the code in the I/O.

Reply:
Moved to util file